contracts/OrcaMemberToken.sol

```diff
@@ -14,14 +17,59 @@ import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
 // only allow for one token per user
 
 contract OrcaMemberToken is ERC1155 {
-    constructor() public ERC1155("ORCA TOKENS FOOL!") {}
+    using SafeMath for uint256;
+    using Address for address;
 
-    function mint(
+    address podManager;
+
+    struct Pod {
+        address creator;
+        uint256 totalSupply;
+    }
+
```

Comment:
can you add something like `uint256[] podIds public` so we can get a list of all pods ?

Reply:
do we also need an array of users within each pod?

Reply:
yah I guess we can add it for connivence for now

Reply:
can we merge this in and I'll do that as later work?

Reply:
I ask because maintaining arrays of users/whatever is actually a good amount of work so I just wanna get this merged

Reply:
ok merging now